composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "mikecao/flight",
+    "name": "s4wny/flight",
```

Comment:
lol, apparently github automatically adds all commits on the master branch to this pull request.

Added that line so I can install my modifed version via composer. Lesson learned: always use a feature branch for pull requests.